api/src/test/java/org/openmrs/module/providermanagement/api/ProviderManagementServiceTest.java

```diff
@@ -3244,6 +3244,61 @@ public void getProvidersQuery_shouldFetchByIdentifierAndNameAndOrderByName()  th
         Assert.assertEquals(new Integer(2), providers.get(1).getId());
         Assert.assertEquals(new Integer(9), providers.get(2).getId());
     }
-
+    
+    @Test (expected = APIException.class)
+    public void getPatientRelationshipForProvider_shouldThrowAPIException() throws Exception {
+    	List<Relationship> actual = providerManagementService.getPatientRelationshipsForProvider(null, null, new Date());
+    	Assert.assertTrue(actual == null || actual.size() == 0);
```

Comment:
This is because the parameters passed in are null?  If so, the test case should say this, ie ... _shouldThrowAPIExceptionIfNoProviderOrRelationshipSpecified.